src/App.js

```diff
@@ -36,14 +36,21 @@ class App extends Component {
   render() {
     return (
       <div className="App">
-        <Header toggleGridView={this.toggleGridView}
-          gridView={this.state.gridView}/>
-        <Route exact path="/" render={()=>  
-          <GridView toggleGridView={this.toggleGridView}
-            movies={this.state.movies}/>} />
-          <Route exact path="/:movieId" render={({match})=>  
-            <SingleMovie id={match.params.movieId} 
-            setGridViewFalse={this.setGridViewFalse}/>} />
+        <Header 
```

Comment:
This is so much more readable, thank you so much for organizing the file like this!